frontend/src/pages/team/Brokers/index.vue

```diff
@@ -91,7 +99,9 @@ export default {
     },
     data () {
         return {
-            loading: true
+            loading: true,
+            brokerState: 'updating',
```

Comment:
```suggestion
            brokerState: '',
```

Default to blank state - with the `pendingStateChange` property set as per other comment - results in the following status (with it animated). I think that looks better than the 'updating' with the animated download arrow.
<img width="50" alt="image" src="https://github.com/user-attachments/assets/27693227-dfcc-4392-9f8a-2f986a220fb0" />